src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/SolutionExplorer_OutOfProc.cs

```diff
@@ -116,8 +116,8 @@ public void CloseFile(ProjectUtils.Project project, string fileName, bool saveFi
         public void SaveFile(ProjectUtils.Project project, string fileName)
             => _inProc.SaveFile(project.Name, fileName);
 
-        public void ReloadProject(ProjectUtils.Project project)
-            => _inProc.ReloadProject(project.Name);
+        public void ReloadProject(string projectRelativePath)
+            => _inProc.ReloadProject(projectRelativePath);
```

Comment:
This should continue to take a `ProjectUtils.Project` and use the `RelativePath` off of that.

The `ProjecetUtils.Project` type exists so that we don't just treat everything as a string, which has tripped us up in the past. We shouldn't go back to that.